src/CloudFoundryApiClient/src/CfApiClient.cs

```diff
@@ -24,12 +24,12 @@ public class CfApiClient : ICfApiClient
         internal const string ListBuildpacksPath = "/v3/buildpacks";
         internal const string DeleteAppsPath = "/v3/apps";
         internal const string ListStacksPath = "/v3/stacks";
+        internal const string LoginInfoPath = "/login"; // the /login endpoint should be identical to the /info endpoint (for CF UAA v 75.10.0)
```

Comment:
Didn't end up needing to turn this into a constant after all, but left it in this PR just because it's nice to follow the convention of the other constant endpoints